addons/block_code/blocks/sounds/play_sound.tres

```diff
@@ -10,7 +10,7 @@ description = "Play the audio stream with volume and pitch"
 category = "Sounds"
 type = 2
 variant_type = 0
-display_template = "Play the sound {name: STRING} with Volume dB {db: FLOAT} and Pitch Scale {pitch: FLOAT}"
+display_template = "play the sound {name: STRING} with Volume dB {db: FLOAT} and Pitch Scale {pitch: FLOAT}"
```

Comment:
```suggestion
display_template = "play the sound {name: STRING} with volume {db: FLOAT} dB and pitch scale {pitch: FLOAT}"
```

`dB` is a unit, it goes after the value.

Reply:
Good point, and I forgot to lower case Volume and Pitch Scale.